### ISSUE TITLE
TextConsole: Arrow keys skip a character sometimes

### ISSUE THREAD

**Issue by TSRBerry** [JoeStrout/Farmtronics#40]
Just a small issue I noticed while playing around with the console.
I tried to change the status color of the bot like this:

```
bot.statusColor = "#FF0000"
```

This works of course, but if I go back and want to change the 0 next to the quotation mark it'll skip over it and the 0 gets selected instead.

**Comment by JoeStrout** [comment]
I'm not able to reproduce this:

![WorksForMe](https://user-images.githubusercontent.com/263365/193418774-a25c59fd-2d7a-4ca3-a019-e7e57578000e.gif)

Any idea what might be different in your case?

**Comment by TSRBerry** [comment]
Oh that's odd, I did exactly that. Not quite sure what was different, but I'll be testing with the console in a bit again.

**Comment by TSRBerry** [comment]
Just managed to reproduce it:
This is what I typed in the console:

```
bot.left
bot.left
bot.statusColor "#00FF00"
```

After getting the error I tried to go up once and saw the last successful command. Then I pressed up a few times more and pressed down to get to the statusColor command. After that I was holding CTRL while using the arrow keys to get to the right location to adjust the syntax. But when I reached it, I let go of CTRL and noticed when I press the arrow keys now the terminal always skips one character.

https://user-images.githubusercontent.com/20988865/193428441-bfbcc7be-1222-4fbc-9846-132e31c2a815.mp4
(I really need something to create gifs)
